src/NuGet.Clients/NuGet.CommandLine/Program.cs

```diff
@@ -310,21 +310,18 @@ private static void RegisterExtensions(AggregateCatalog catalog, IEnumerable<str
 
         private static void SetConsoleInteractivity(IConsole console, Command command)
         {
-            // Global environment variable to prevent the exe for prompting for credentials
-            string globalSwitch = Environment.GetEnvironmentVariable("NUGET_EXE_NO_PROMPT");
-
-            // When running from inside VS, no input is available to our executable locking up VS.
-            // VS sets up a couple of environment variables one of which is named VisualStudioVersion.
-            // Every time this is setup, we will just fail.
-            // TODO: Remove this in next iteration. This is meant for short-term backwards compat.
-            string vsSwitch = Environment.GetEnvironmentVariable("VisualStudioVersion");
+            // Apply command setting
```

Comment:
are we documenting this somewhere?

Reply:
Was it documented to start with?

Reply:
oh shoot,  i didn't see the left hand side of the PR, thought you added these